src/plone/restapi/tests/test_dxcontent_serializer.py

```diff
@@ -88,8 +89,10 @@ def setUp(self):
         self.portal.doc1.modification_date = DateTime("2015-04-27T10:24:11+00:00")
         IMutableUUID(self.portal.doc1).set("30314724b77a4ec0abbad03d262837aa")
 
-    def serialize(self):
-        serializer = getMultiAdapter((self.portal.doc1, self.request), ISerializeToJson)
+    def serialize(self, obj=None):
```

Comment:
I already commented on this in the other PR. Can you elaborate on this? If you need a more specific serializer, then you call the multiadapter locally in your test, right? What is the reason to pervert the original implementation of this helper?

Reply:
answered there.. copying here:
```
This method is hardcoded to self.portal.doc1 and I need to serialize other objects.

Sure, it is intended to have this condition because all other tests expect to be this object.
```